pypeit/core/wavecal/wvutils.py

```diff
@@ -185,11 +185,12 @@ def get_wave_grid(waves, masks=None, wave_method='linear', iref=0, wave_grid_min
 
     Returns:
         :obj:`tuple`: Returns two `numpy.ndarray`_ objects and a float:
-            - ``wave_grid``: New wavelength grid, not masked
-            - ``wave_grid_mid``: New wavelength grid evaluated at the centers of
+            - ``wave_grid``: (ndarray, (ngrid +1,)) New wavelength grid, not masked
```

Comment:
Maybe add a note that wave_grid constitutes the set of bin edges, whereas wave_grid_mid is the set of bin centres, hence the different number of pixels.

Reply:
Done.